drivers/sensor/max17262/max17262.c

```diff
@@ -178,26 +175,19 @@ static int max17262_channel_get(const struct device *dev,
  * @param dev MAX17262 device to access
  * @return 0 if successful, or negative error code from I2C API
  */
-static int max17262_sample_fetch(const struct device *dev,
-				 enum sensor_channel chan)
+static int max17262_sample_fetch(const struct device *dev, enum sensor_channel chan)
 {
 	struct max17262_data *data = dev->data;
 	struct {
 		int reg_addr;
 		int16_t *dest;
 	} regs[] = {
-		{ VCELL, &data->voltage },
-		{ AVG_CURRENT, &data->avg_current },
-		{ ICHG_TERM, &data->ichg_term },
-		{ REP_SOC, &data->state_of_charge },
-		{ INT_TEMP, &data->internal_temp },
-		{ REP_CAP, &data->remaining_cap },
-		{ FULL_CAP_REP, &data->full_cap },
-		{ TTE, &data->time_to_empty },
-		{ TTF, &data->time_to_full },
-		{ CYCLES, &data->cycle_count },
-		{ DESIGN_CAP, &data->design_cap },
-		{ COULOMB_COUNTER, &data->coulomb_counter },
+		{VCELL, &data->voltage},          {AVG_CURRENT, &data->avg_current},
+		{ICHG_TERM, &data->ichg_term},    {REP_SOC, &data->state_of_charge},
+		{INT_TEMP, &data->internal_temp}, {REP_CAP, &data->remaining_cap},
+		{FULL_CAP_REP, &data->full_cap},  {TTE, &data->time_to_empty},
+		{TTF, &data->time_to_full},       {CYCLES, &data->cycle_count},
+		{DESIGN_CAP, &data->design_cap},  {COULOMB_COUNTER, &data->coulomb_counter},
```

Comment:
This makes no sense at all, then if any of the left side changes it's likely to cause the whole block to be realigned, causing even more unnecessary diffs, potentially making other changes harder to spot in code reviews etc. I'd suggest adding a `clang-format off` around this if you really want to reformat the file but really I don't think we should attach file-wide reformat next to bugfixes. I could understand if the file had a major refactoring but this is just adding noise.

Reply:
I have the reformatting as a separate commit and could drop that if you want.

Reply:
Up to you whether to keep it, I would not bother unless I was doing heavy rework on the driver, but if you keep it I'd ask you to block this one off.

Reply:
Excluded this portion from formatting.